src/bin_scripts/ic-pack.ts

```
@@ -10,372 +1,384 @@
+            const packageName = isProductionEnv
+              ? `@${packageScope}/${name}_client`
+              : `@${packageScope}/${name}_${canisterEnv}_client`;
+            const npm_client_input: PackNpmClientInput = {
+              did_file_path: did_path,
+              target_dir_path: `${npm_dir}/client/${name}`,
+              name: packageName,
+              version: buildContext.icPackInput.version
             };
+            pack_npm_client(npm_client_input);
+          }
         }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)